lib/src/config.rs

```diff
@@ -215,6 +217,28 @@ pub struct RepoEncryption {
     pub nonce: secretbox::Nonce,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+/// Chunking Algorithms supported by rdedup
+pub enum ChunkingAlogrithm {
+    Bup,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+/// RepoChunking is the configuration of the chunking algorithm and parameters for the repo
+pub struct RepoChunking {
+    pub algorithm: ChunkingAlogrithm,
+    pub size: u32, // size is generic enough to work with different algorithms
```

Comment:
It can vary per implementation. So if we went with fixed chunking it could be 192K which is not log2